docs/installation/frameworks/vuejs-v2.md

```diff
@@ -238,13 +238,85 @@ Finally, import the build in your application:
 
 Integrating the rich text editor from source allows you to use the full power of {@link framework/index CKEditor 5 Framework}.
 
+### Vite
+
+This guide assumes that you use `create-vite` as your boilerplate. To get started with Vite and Vue 2, run the command below.
+
+```bash
+# npm 6.x
+npm init vite@latest ckeditor5-vue2-example --template vue
+
+# npm 7+, extra double-dash is needed:
+npm init vite@latest ckeditor5-vue2-example -- --template vue
+```
+
+This command will install and execute `create-vite`, the official project scaffolding tool for Vite.
+
+#### Installing necessary packages
+
+You need some packages to use CKEditor from source with Vue 2 and Vite: the official CKEditor Vue component, the CKEditor Vite plugin, and a theme. Also, Vite assumes you'll use Vue 3, so you need to fix that.
+
+First, uninstall Vue 3 and `@vitejs/plugin-vue`.
+
+```bash
+npm uninstall vue @vitejs/plugin-vue
+```
+
+Then, install the Vue 2 package and the Vue 2 plugin for Vite.
+
+```bash
+npm install --save vue@2 @vitejs/plugin-vue2
+```
+
+<info-box>
+	Using the Vite plugin to build CKEditor 5 from the source in Vite is still in the experimental phase. We encourage you to test it and give us feedback. To read more about integration with Vite or its limitations, check the {@link installation/advanced/integrating-from-source-vite Integrating from source with Vite} guide.
+</info-box>
```

Comment:
Instead of writing a guide for converting Vue 3 app back to Vue 2, let's assume that the user has Vite + Vue 2 application with `@vitejs/plugin-vue2` already configured. 

In my opinion, it's not expected steps and if someone is starting a new project they should use Vue 3. Other than changing packages and vite config, there are also some breaking changes related to Vue usage in source files and that needs to be also changed. We do not need to cover that.

When writing this part of the guide, let's assume that we are writing it for someone who has Vue 2 migrated from Webpack to Vite, and now they need some information on what additional packages they need to install and how to use them to have the editor built from source in their Vite+Vue2 app.